PluralKit.Bot/Proxy/ProxyService.cs

```diff
@@ -338,6 +389,9 @@ Task SaveMessageInDatabase()
 
         async Task DeleteProxyTriggerMessage()
         {
+            if (!deletePrevious)
```

Comment:
what is this for?

Reply:
the task in HandleProxyExecuteActions will delete the reproxied message if we delete the trigger, hence some of the duplication of message deletion stuff & this check to return early in that task